__tests__/topics.test.js

```diff
@@ -0,0 +1,28 @@
+const app = require('../app.js');
+const request = require('supertest');
+const db = require('../db/connection.js');
+const seed = require('../db/seeds/seed.js')
+const topicData = require('../db/data/test-data/index.js');
+
+beforeEach(() => seed(topicData));
+afterAll(() => db.end());
+
+describe('api/topics', () => {
+    test('GET: 200, an array of topic objects with properties "slug" and "description"', () => {
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then((response) => {
+            expect(response.body.topics).toEqual(expect.any(Array));
+            expect.arrayContaining(['slug', 'description']);
+        })
+    })
+    test('GET: 404, returns an error when a route does not exist', () => {
```

Comment:
Does this error relate to the end point /api/topics? If not, it should be in its own describe block as it does not form testing for this endpoint.